packages/frontend-2/assets/css/tailwind.css

```diff
@@ -51,15 +52,18 @@
   .caption {
     @apply text-xs;
   }
-}
 
-/**
- * Ensuring ring offset color always defaults to bg color
- */
-*,
-[type='checkbox']:focus,
-[type='radio']:focus {
-  @apply ring-offset-background;
+  .default-width {
```

Comment:
Can you elaborate on what this is? Preferably `tailwind.css` should only contain either global styles or globally re-usable classes that will be used often, and I'm not sure if it's the case for "default-width". The naming is confusing as well - the default width of what? The body? An element inside the body? A column?

Reply:
second conclusion: we need it, and let's name it `layout-single-column`